x-pack/plugins/viz_editor/public/pseudo_plugins/scatter_chart_plugin/scatter_chart_plugin.tsx

```diff
@@ -89,7 +92,7 @@ function toExpression(visModel: ScatterChartVisModel, mode: 'edit' | 'view' | 'p
     "scales": [
       {
         "name": "x",
-        "type": "linear",
+        "type": "${xScaleType}",
```

Comment:
Does this render for you? For me it looks like this:
<img width="1139" alt="Screenshot 2019-04-05 at 11 32 08" src="https://user-images.githubusercontent.com/1508364/55618320-7e77c480-5796-11e9-9bd8-9eda62897d2b.png">

Probably because the date ISO strings are not mapped to something vega can work with

Reply:
Yeah, that's right. There's probably more work needed to solve this